.gitignore

```diff
@@ -11,6 +11,7 @@ simulator
 *_*.tigerbeetle.*
 zig/
 zig-cache/
+zig-global-cache/
```

Comment:
Can we set the global cache as `zig-cache/global`? It's a folder less to care about.

Reply:
Boa ideia!